project/webapp/static/js/thread.js

```diff
@@ -1555,13 +1513,23 @@ cw.ThreadView = BB.View.extend({
     },
 
     renderBodyContents: function () {
+        if (!this.is_draft) {
```

Comment:
Consider using the template layer to show/hide elements based on simple conditionals:

```html
{{# if (!is_draft) { }}
  ...

```